docker/compose/docker-compose_hadoop284_hive233_spark231.yml

```diff
@@ -0,0 +1,215 @@
+version: "3.3"
+
+services:
+
+  namenode:
+    image: varadarb/hudi-hadoop_2.8.4-namenode:latest
```

Comment:
Is this correct ? Should the username be here ? @bvaradar

Reply:
@n3nash :
I am not aware of any headless docker hub account . I have created a ticket to provide more context and assigned to @vinothchandar  

https://github.com/uber/hudi/issues/469

Once the ticket is resolved, we can replace the image locations in Dockerfile, pom.xml and in compose scripts.